src/app/containers/TopStories/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useContext } from 'react';
+import { arrayOf, shape } from 'prop-types';
+import SectionLabel from '@bbc/psammead-section-label';
+import styled from 'styled-components';
+import { StoryPromoLi, StoryPromoUl } from '@bbc/psammead-story-promo-list';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+
+import topStories from '#pages/StoryPage/topStories.json';
+import { storyItem } from '#models/propTypes/storyItem';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { GridItemConstrainedLarge } from '#lib/styledGrid';
+import StoryPromo from '../StoryPromo';
+
+const TopStoriesWrapper = styled(GridItemConstrainedLarge).attrs({
+  as: 'section',
+  role: 'region',
+  'aria-labelledby': 'top-stories-heading',
+})`
+  margin-bottom: ${GEL_SPACING_DBL};
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    margin-bottom: ${GEL_SPACING_TRPL};
+  }
+`;
+
+const StyledSectionLabel = styled(SectionLabel)`
+  margin-top: 0;
+`;
+
+const TopStories = ({ content }) => {
+  const { script, service, dir } = useContext(ServiceContext);
+
+  if (!content || !content.length) return null;
+
+  return (
+    <TopStoriesWrapper>
+      <StyledSectionLabel
+        script={script}
+        service={service}
+        dir={dir}
+        labelId="top-stories-heading"
+      >
+        Top Stories
+      </StyledSectionLabel>
+
+      <StoryPromoUl>
+        {content.map(item => (
+          <StoryPromoLi key={item.id || item.uri}>
+            <StoryPromo item={item} dir={dir} displayImage={false} />
+          </StoryPromoLi>
+        ))}
+      </StoryPromoUl>
+    </TopStoriesWrapper>
+  );
+};
+
+TopStories.propTypes = {
+  content: arrayOf(shape(storyItem)),
+};
+
+TopStories.defaultProps = {
+  content: topStories, // @TODO: rm this
```

Comment:
Could this TODO include an issue where the faked fixture data will be replaced with the real data endpoint?

Reply:
Done!